README.md

```diff
@@ -21,6 +21,29 @@ Or install it yourself as:
 $ gem install perilune
 ```
 
+## Configuration
+You can use diffrent queue name for cron job:
+
+```ruby
+Periline::Configuration do |config|
+  config.queue_name = 'your_queue_name'
+end
+```
+
+> Note - Perilune is using *default* as queue name.
+
+You can configure the driver for Trifle::Stats :
+
+```ruby
+Periline::Configuration do |config|
+  config.trifle_stat_driver = ::Trifle::Stats::Driver::Redis.new
```

Comment:
You can just name this `stats_driver`. Also if gem is called `Trifle::Stats` and you wanna use that as a variable name, make sure it is kinda matching the name. You skipped pluralization in `stat`.